nautobot/ipam/filters.py

```diff
@@ -489,7 +498,7 @@ def filter_device(self, queryset, name, value):
             return queryset.none()
         interface_ids = []
         for device in devices:
-            interface_ids.extend(device.vc_interfaces.values_list("id", flat=True))
+            interface_ids.extend(device.all_interfaces.values_list("id", flat=True))
```

Comment:
@HanlinMiao Any chance this should also be reverted as part of #6046?

Reply:
`all_interfaces` is useful for specific purposes (getting interfaces that belong to this device + its contained modules, not including virtual-chassis) but I wonder if we should rename it to something like `contained_interfaces` for clarity.

Reply:
@timizuoebideri1 this change too.

Reply:
#6064